src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -4015,13 +4015,16 @@ extern "C" CLR_BOOL QCALLTYPE SfiNext(StackFrameIterator* pThis, uint* uExCollid
             // Check if there are any further managed frames on the stack or a catch for all exceptions in native code (marked by
```

Comment:
I got a bit confused by `invalidRevPInvoke` local variable name. I am not sure what's invalid about it.

Would `isReversePInvoke` be a better name for it?

Reply:
That's a historical name that was created when the `Interop::ManagedToNativeExceptionCallback` was introduced. It is confusing to me too. I am planning to restructure this code a bit as part of a fix to the #116676, so I will address it as part of that.